test/smp/regression/adp/cases/dsd_uds_512kb_3k_contexts_throughput/experiment.yaml

```diff
@@ -0,0 +1,28 @@
+optimization_goal: ingress_throughput
+erratic: false
+
+target:
+  name: agent-data-plane
+  command: /usr/local/bin/agent-data-plane
+  cpu_allotment: 4
+  memory_allotment: 2GiB
+
+  environment:
+    DD_API_KEY: foo00000001
+    DD_HOSTNAME: smp-regression
+    DD_DD_URL: http://127.0.0.1:9091
+
+    # Set the context limit in the aggregator to right above 3K, which matches the maximum number of contexts we expect
+    # to generate. We essentially don't want the aggregator to be a limiting factor.
+    DD_AGGREGATE_CONTEXT_LIMIT: "3100"
```

Comment:
Is there a reason why we don't set `dogstatsd_cached_contexts_limit` as well? For my own experimenting, I was noticing that `adp.context_resolver_active_contexts` was higher than the number of contexts configured in my lading.yaml until I set the limit to match. (But also, I maybe be doing something wrong down stream 🤔 )

Reply:
No _good_ reason to not set it, but the default is high enough that we never hit the limit... but the defaults for the aggregator _are_ low enough that we do hit it.

Makes me think that we should actually change the defaults of the aggregator to match the context resolver. 🤔